### ISSUE TITLE
[Bug]: [Snappi] the value of num_iterations is getting over-written in multidut_helper file

### ISSUE THREAD

**Issue by amitpawar12** [sonic-net/sonic-mgmt#12852]
### Issue Description

Testcase - tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py

```
num_iterations = 1
ip_pkts_list = run_ecn_test(api=snappi_api,
-------------------- curtailed output ------------------------
                                iters=num_iterations,
                                snappi_extra_params=snappi_extra_params)
```

Multidut_helper script - tests/snappi_tests/multidut/ecn/files/multidut_helper.py
However, in the multidut_helper file, the value of iters is getting overwritten:
`    iters = snappi_extra_params.test_iterations`

Thus no matter, what you set the num_iterations to, the multidut_helper will run the test once. 

There is check in the original test to check the iterations vs the packet-capture size and test fails there.

### Results you see

```
18:57:23 test_multidut_red_accuracy_new.test_red_ L0103 INFO   | Running ECN red accuracy test for 2 iterations
18:57:48 multidut_helper.run_ecn_test             L0194 INFO   | Running 1 iteration(s)
18:57:48 multidut_helper.run_ecn_test             L0196 INFO   | Running iteration 0
```

### Results you expected to see

```
18:57:23 test_multidut_red_accuracy_new.test_red_ L0103 INFO   | Running ECN red accuracy test for 2 iterations
18:57:48 multidut_helper.run_ecn_test             L0194 INFO   | Running 2 iteration(s)
18:57:49 multidut_helper.run_ecn_test             L0196 INFO   | Running iteration 0
:
18:58:48 multidut_helper.run_ecn_test             L0196 INFO   | Running iteration 1
```

### Is it platform specific

generic

**Comment by amitpawar12** [comment]
@kamalsahu0001

**Comment by selldinesh** [comment]
@amitpawar12 Fix is available in https://github.com/sonic-net/sonic-mgmt/pull/12853

**Comment by amitpawar12** [comment]
Thanks @selldinesh.

**Comment by amitpawar12** [comment]
Fix for this issue is available in #12853
